tools/storage-advisor/src/s3Issues.go

```diff
@@ -44,72 +85,110 @@ func listS3Buckets(client *s3.Client) ([]string, error) {
 }
 
 type Bucket struct {
-	Name            string `json:"name"`
-	Region          string `json:"region"`
-	Ownership       string `json:"ownership"`
-	OwnershipIssue  string `json:"ownershipIssue"`
-	Versioning      string `json:"versioning"`
-	VersioningIssue string `json:"versioningIssue"`
-	Inventory       string `json:"inventory"`
-	Logging         string `json:"logging"`
-	Encryption      string `json:"encryption"`
-	EncryptionIssue string `json:"encryptionIssue"`
+	Name                   string `json:"name" bson:"name"`
+	Region                 string `json:"region" bson:"region"`
+	Ownership              string `json:"ownership" bson:"ownership"`
+	OwnershipIssue         string `json:"ownershipIssue" bson:"ownershipIssue"`
+	OwnershipIssueDetails  string `json:"ownershipIssueDetails" bson:"ownershipIssueDetails"`
+	Versioning             string `json:"versioning" bson:"versioning"`
+	VersioningIssue        string `json:"versioningIssue" bson:"versioningIssue"`
+	VersioningIssueDetails string `json:"versioningIssueDetails" bson:"versioningIssueDetails"`
+	Inventory              string `json:"inventory" bson:"inventory"`
+	InventoryIssue         string `json:"inventoryIssue" bson:"inventoryIssue"`
+	InventoryIssueDetails  string `json:"inventoryIssueDetails" bson:"inventoryIssueDetails"`
+	Logging                string `json:"logging" bson:"logging"`
+	LoggingIssue           string `json:"loggingIssue" bson:"loggingIssue"`
+	LoggingIssueDetails    string `json:"loggingIssueDetails" bson:"loggingIssueDetails"`
+	Encryption             string `json:"encryption" bson:"encryption"`
+	EncryptionIssue        string `json:"encryptionIssue" bson:"encryptionIssue"`
+	EncryptionIssueDetails string `json:"encryptionIssueDetails" bson:"encryptionIssueDetails"`
+
+	UsableInventory string `json:"usableInventory" bson:"usableInventory"`
```

Comment:
I'd like to point out that there are unused `bson` field tags. They do no harm though other than making me question their utility.